Lib/test/test_os.py

```diff
@@ -4206,6 +4208,21 @@ def test_attributes(self):
             entry = entries['symlink_file.txt']
             self.check_entry(entry, 'symlink_file.txt', False, True, True)
 
+    @unittest.skipIf(sys.platform != 'win32', "Can only test junctions with creation on win32.")
+    def test_attributes_junctions(self):
+        dirname = os.path.join(self.path, "tgtdir")
+        os.mkdir(dirname)
+
+        import _winapi
+        try:
+            _winapi.CreateJunction(os.path.join(self.path, "srcjunc"), dirname)
```

Comment:
```suggestion
            _winapi.CreateJunction(dirname, os.path.join(self.path, "srcjunc"))
```

`CreateJunction()` uses the same parameter order as `os.symlink()`. The first parameter is the target path. The second parameter is the path of the junction to be created. Passing the existing directory `dirname` as the second argument causes the call to raise `FileExistsError`, and the test gets skipped.